examples/go/tagging/handler/handler.go

```diff
@@ -141,24 +138,20 @@ func connect(ctx context.Context, vca vCenterAuth) *Clients {
 		Path:   "sdk",
 	}
 
-	log.Println("Get govmomi client.")
 	govc, err := govmomi.NewClient(ctx, &u, true)
 	if err != nil {
 		log.Println(err)
 	}
 	c.Govmomi = govc
 
-	log.Println("Login to govmomi client")
 	u.User = url.UserPassword(vca.User, vca.Password)
 
 	err = c.Govmomi.Login(ctx, u.User)
 	if err != nil {
-		log.Printf("ERROR: %v", err)
+		log.Print(err)
```

Comment:
not sure if `log.Println()` would be better to not mess up console output (newlines)

Reply:
Removed Govmomi.Login by adding User and Password info into the url.URL.